azure-quantum/azure/quantum/optimization/problem.py

```diff
@@ -59,12 +70,14 @@ def __init__(
         terms: Optional[List[TermBase]] = None,
         init_config: Optional[Dict[str, int]] = None,
         problem_type: ProblemType = ProblemType.ising,
+        serialization_type: Optional[str] = None
```

Comment:
I feel this one should be an enum (like `ProblemType`) as opposed to a free-form string, as the code will take specific actions based on its value.